src/plugins/pairing_acceleration/pairing_variations.py

```diff
@@ -83,50 +79,50 @@ def compute_virtual_points(
         tournament: Tournament,
         player: Player,
         at_round: int,
+    ) -> float:
+        return ProgressiveSwissVariation._compute_progressive_virtual_points(
+            tournament, player, at_round
+        )
+
+    @staticmethod
+    def _compute_progressive_virtual_points(
+        tournament: Tournament,
+        player: Player,
+        at_round: int,
+        virtual_draw_points_per_real_draw_point: int = 3,
```

Comment:
It's the other way around: add a virtual draw per so many real draw equivalent.

Reply:
More explicitly: 3 virtual draw points per real draw point (in a 1-0.5-0) system would mean 1.5 virtual point per 0.5 points.